src/main/java/fr/openent/homeworkAssistance/service/IResourcesService.java

```diff
@@ -0,0 +1,18 @@
+package fr.openent.homeworkAssistance.service;
+
+import fr.openent.homeworkAssistance.models.FeaturedResources;
+import io.vertx.core.Future;
+import io.vertx.core.json.JsonArray;
+import org.entcore.common.user.UserInfos;
+
+import java.util.List;
+
+public interface IResourcesService {
+
+    /**
+     * Get resources from Mediacentre to display
+     * @param user User who request resources
+     * @return Future with JsonArray of resources
```

Comment:
Maintenant c'est plutôt un truc du genre :
```suggestion
     * Get resources from Mediacentre to display
     * @param user The {@link UserInfos} of the connected user
     * @return A {@link Future} containing {@link List<FeaturedResource>} with the ressources to display to the connected user
```